README.md

```diff
@@ -1,127 +1,50 @@
 Floodlight OpenFlow Controller (OSS)
 ====================================
 
-Build Status:
--------------
+Attention!
+----------
 
-[![Build Status](https://travis-ci.org/floodlight/floodlight.svg?branch=master)](https://travis-ci.org/floodlight/floodlight)
-
-Floodlight Wiki
----------------
+As of August 2018, the Floodlight mailing list has moved! Please see **Documentation and Support** below for up-to-date support information.
```

Comment:
It's redundant, but we should probably include where it's moved to here. Something like "the Floodlight mailing list has moved to floodlight@groups.io". Also, I think with markdown you can directly link to another header

Reply:
Yep, that's a good idea